zilencer/lib/stripe.py

```diff
@@ -122,7 +122,7 @@ def do_create_customer_with_payment_source(user: UserProfile, stripe_token: str)
     realm = user.realm
     stripe_customer = stripe.Customer.create(
         description="%s (%s)" % (realm.string_id, realm.name),
-        metadata={'realm_id': realm.id, 'realm_str': realm.string_id},
+        metadata={'realm_id': realm.id, 'realm_str': realm.string_id, 'email': user.email},
```

Comment:
This should be an argument to create: https://stripe.com/docs/api#customer_object-email